scripts/velero.sh

```diff
@@ -27,14 +27,26 @@ then
       --snapshot-location-config region=$VELERO_REGION \
       --secret-file keys/credentials-velero
   velero snapshot-location create vsl-vsphere --provider velero.io/vsphere
-else
-  velero install \
+elif [ -z "$AWS_SESSION_TOKEN" ]; 
```

Comment:
Can't assume user is aws as ias.  Must test for params.yaml .iaas = aws

Reply:
we actually are checking for IAAS vsphere in the previous block.  Unfortunately using vsphere with the s3/session tokens won't work.